ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -479,6 +486,16 @@ var _ = Context("Inside the default namespace", func() {
 			Eventually(
 				getClusterState(ctx, namespace, rayCluster.Name),
 				time.Second*3, time.Millisecond*500).Should(Equal(rayv1.Suspended))
+			if withConditionEnabled {
+				Eventually(
+					func() rayv1.RayClusterConditionType {
```

Comment:
This anonymous function is defined three times in this PR. Could we define it once and reuse it?